h2o-genmodel/src/main/java/hex/genmodel/algos/targetencoder/TargetEncoderMojoModel.java

```diff
@@ -14,12 +14,23 @@ public static double computeLambda(long nrows, double inflectionPoint, double sm
   public static double computeBlendedEncoding(double lambda, double posteriorMean, double priorMean) {
     return lambda * posteriorMean + (1 - lambda) * priorMean;
   }
+  
+  static Map<String, Integer> name2Idx(String[] columns) {
+    Map<String, Integer> nameToIdx = new HashMap<>(columns.length);
+    for (int i = 0; i < columns.length; i++) {
+      nameToIdx.put(columns[i], i);
+    }
+    return nameToIdx;
+  }
+
 
   public final Map<String, Integer> _columnNameToIdx;
   public Map<String, Boolean> _teColumn2HasNAs; // tells if a given encoded column has NAs
   public boolean _withBlending;
   public double _inflectionPoint;
   public double _smoothing;
+  public List<ColumnsToSingleMapping> _inencMapping;
```

Comment:
@mn-mikke is it ok to have it in the MojoModel or would you prefer it in `TargetEncoderPrediction`? as a result of `EasyPredictModelWrapper.predictTargetEncoding`?

Reply:
@sebhrusen It could stay here, but what mapping I'm supposed to use? `_inencMapping` or `_inoutMapping`?

Reply:
@mn-mikke thanks for pointing that out: I underlined the wrong attribute for you: I'm pretty sure that you need only `_inoutMapping` as its name suggests.